### ISSUE TITLE
Broken link in the Assignment section(number 2)

### ISSUE THREAD

**Issue by Bright-Anyawe** [TheOdinProject/curriculum#28669]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

The second link(number 2) in the assignment section does not provide any information relating to the topic at hand

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/javascript-working-with-remotes

### (Optional) Discord Name

Bryt55

**Comment by Bidrift** [comment]
I have checked it. It seems to be working fine, it redirects you to the introduction of the article, you should just press "[Let's get started, shall we?](https://think-like-a-git.net/sections/about-this-site.html)"

**Comment by MaoShizhong** [comment]
I think we can make our assignment step clearer as to exactly what we want people to do.

## Acceptance criteria

- [ ] Assignment step 2's link text is changed to `"Think Like (a) Git"`
- [ ] Change the href for the 2nd assignment's link to `https://think-like-a-git.net/sections/about-this-site.html`
- [ ] Make it clear that the assignment is to read the whole thing e.g. reword to `Read the entirety of [Think Like (a) Git](...).`

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by eGOist-sys** [comment]
i would like to work on this issue!

**Comment by MaoShizhong** [comment]
All yours @eGOist-sys

**Comment by blue0206** [comment]
Thanks for the quick response and for passing this along!

**Comment by JustWaveThings** [comment]
> Hi @MaoShizhong , is anyone actively working on this? If not, I'd like to take it on. (Sorry for the ping)  

All yours @blue0206. Please reach out here if you run into any problems!

**Comment by blue0206** [comment]
Thanks! I'll take it from here. I'll let you know if I encounter any issues.

**Comment by blue0206** [comment]
@JustWaveThings I have made the fixes and created a PR #29427 
I've followed along the acceptance criteria. Please let me know if any adjustments are needed. Thanks!
